frigate/detectors/plugins/edgetpu_tfl.py

```diff
@@ -44,6 +44,7 @@ def __init__(self, detector_config: EdgeTpuDetectorConfig):
                 model_path=detector_config.model.path,
                 experimental_delegates=[edge_tpu_delegate],
             )
+            self.order = detector_config.model.order
```

Comment:
I don't think there is a reason to set `self.order` every time, just set it once in init